eng/Versions.props

```diff
@@ -202,6 +201,7 @@
     <!-- Packages developed by @aspnet, but manually updated as necessary. -->
     <LibuvPackageVersion>1.10.0</LibuvPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.6</MicrosoftAspNetWebApiClientPackageVersion>
+    <MonoWebAssemblyInteropPackageVersion>5.0.0-preview3.1.20079.1</MonoWebAssemblyInteropPackageVersion>
```

Comment:
Had to make this PR locally because this package is no longer produced by Extensions and was causing issues with the bot. I've asked @pranavkm to clean up the `MonoWebAssemblyInteropPackageVersion` usage in the repo.